nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -132,46 +134,46 @@ def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
 
     @staticmethod
     @abstractmethod
-    def get_bias_value(model: TModel, node: NNCFNode) -> np.ndarray:
+    def get_bias_value(model: TModel, bias_node: NNCFNode) -> np.ndarray:
         """
         Returns bias value in the NumPy format of provided node.
 
         :param model: Backend-specific model for the initializer finding.
-        :param node: Node of NNCFGraph with bias value.
+        :param bias_node: Node of NNCFGraph with bias value.
```

Comment:
Need more details here. I think this docstring is not correct.

Reply:
Which types of details do you mean?